target/linux/ramips/dts/mt7621_mikrotik_routerboard-750gr3.dts

```diff
@@ -35,133 +28,12 @@
 			gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
 		};
 	};
-
-	keys {
-		compatible = "gpio-keys";
-
-		mode {
-			label = "mode";
-			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
-			linux,code = <BTN_0>;
-		};
-
-		reset {
-			label = "reset";
-			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
-			linux,code = <KEY_RESTART>;
-		};
-	};
-
-	beeper {
-		compatible = "gpio-beeper";
-		gpios = <&gpio 15 GPIO_ACTIVE_HIGH>;
-	};
-
-	gpio_export {
-		compatible = "gpio-export";
-		#size-cells = <0>;
-
-		usb_power {
-			gpio-export,name = "usb_power";
-			gpio-export,output = <1>;
-			gpios = <&gpio 12 GPIO_ACTIVE_HIGH>;
-		};
-	};
-};
-
-&spi0 {
-	status = "okay";
-
-	flash@0 {
-		compatible = "jedec,spi-nor";
-		reg = <0>;
-		spi-max-frequency = <20000000>;
-
-		partitions {
-			compatible = "fixed-partitions";
-			#address-cells = <1>;
-			#size-cells = <1>;
-
-			partition@0 {
-				label = "RouterBoot";
-				reg = <0x0 0x40000>;
-				read-only;
-				compatible = "mikrotik,routerboot-partitions";
-				#address-cells = <1>;
-				#size-cells = <1>;
-
-				partition@0 {
-					label = "bootloader1";
-					reg = <0x0 0x0>;
-					read-only;
-				};
-
-				hard_config: hard_config {
-					read-only;
-				};
-
-				partition@10000 {
-					label = "bootloader2";
-					reg = <0x10000 0xf000>;
-					read-only;
-				};
-
-				soft_config {
-				};
-
-				partition@30000 {
-					label = "bios";
-					reg = <0x30000 0x1000>;
-					read-only;
-				};
-			};
-
-			partition@40000 {
-				compatible = "mikrotik,minor";
-				label = "firmware";
-				reg = <0x040000 0xfc0000>;
-			};
-		};
-	};
-};
-
-&switch0 {
-	ports {
-		port@0 {
-			status = "okay";
-			label = "wan";
-		};
-
-		port@1 {
-			status = "okay";
-			label = "lan2";
-		};
-
-		port@2 {
-			status = "okay";
-			label = "lan3";
-		};
-
-		port@3 {
-			status = "okay";
-			label = "lan4";
-		};
-
-		port@4 {
-			status = "okay";
-			label = "lan5";
-		};
-	};
 };
 
 &state_default {
 	gpio {
-		/* via gpio7 (uart3 group) the PoE status can be read */
+		/* on 760iGS via gpio7 (uart3 group) the PoE status can be read */
```

Comment:
Yes, I agree.
If @adrianschmutzler is happy with this shared in DTSI, it would be there. I did not get a final answer on that.
I wanted to show that this (PoE out) is only relevant to 760igs, and not change the existing value without testing on the 750gr3.

Should I just move it back to the shared DTSI?

Reply:
Well, I don't remember what the previous discussion was, but I probably said that the state_default should be moved to the DTS files, as the leds are defined there, and they could at least in principle be different.
Now it seems like we actually have different gpios used for the two devices, to having the state_default node in DTS makes even more sense to me.

Reply:
Okay,
Do you want a change here to be able to merge?

> 760 explicitly needs to have uart3 set to gpio mode.
> On the 750gr3, these lines are unused.
From https://github.com/openwrt/openwrt/pull/3103#discussion_r439734018

Thanks